frappe/core/doctype/system_settings/system_settings.js

```diff
@@ -17,6 +17,11 @@ frappe.ui.form.on("System Settings", {
 			},
 		});
 
+		frappe.xcall("frappe.apps.get_apps").then((r) => {
```

Comment:
Should this be in the bootinfo instead? Might help in other use-cases

Reply:
`get_apps` method is not only getting static data from hooks.py but also check permission for each app. Adding it to boot doesn't make sense